Wikipedia/Code/UIViewController+WMFArticlePresentation.h

```diff
@@ -0,0 +1,24 @@
+
+#import <UIKit/UIKit.h>
+#import "WMFAnalyticsLogging.h"
+#import "WMFArticleViewController.h"
+
+@class MWKTitle, MWKDataStore;
+@class WMFTableOfContentsViewController;
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface UIViewController (WMFArticlePresentation)
+
+- (void)wmf_pushArticleWithTitle:(MWKTitle*)title dataStore:(MWKDataStore*)dataStore restoreScrollPosition:(BOOL)restoreScrollPosition animated:(BOOL)animated;
```

Comment:
@montehurd these are not new methods - this category was just moved from the browser view class that was deleted. Check out the deleted part of the diff to see.

(No behavior was changed in this diff… just the browser code path and tweak were removed)